src/app/widget-builder/components/theme-edit/themes.ts

```diff
@@ -0,0 +1,26 @@
+export const themes = [
+  {
+    name: 'Bram\'s paradise',
+    description: 'a blend of peaceful pastel colours',
+    preview: 'paradise_circus.jpg',
+    stylesheet: 'paradise-circus.css'
+  },
+  {
+    name: 'BILL',
+    description: 'stijl je agenda zoals de BILL jongerenagenda',
+    preview: 'bill.png',
+    stylesheet: 'paradise-circus.css'
+  },
+  {
+    name: 'Indian Summer',
+    description: 'cool',
+    preview: 'indian_summer.png',
+    stylesheet: 'paradise-circus.css'
+  },
+  {
+    name: 'Smooth operator',
+    description: 'Corneel wille\'s smooth theme',
+    preview: 'smooth-operator.jpg',
+    stylesheet: 'smooth-operator.css'
+  }
+]
```

Comment:
If we merge this PR now wouldn't these test themes become available on production? Same for the preview images.

Reply:
Jup there is a subtask for creating the actual themes.
For now we can use these dummy themes. We will replace them once the actual themes are ready.

Reply:
But now that you merged it we have to hurry to finish that subtask because we cannot deploy to production now in case we need to fix a bug or something?